src/front/app/features/main/routes.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Route, IndexRoute } from 'react-router';
+import MainScreen from './MainScreen';
+import Awards from './awards';
+import Updates from './updates';
+import Profile from './profile';
+import ChallengeScreenContainer from './challenges/components/ChallengeScreenContainer';
+
+export default () => (
+    <Route path="main" component={MainScreen} >
+        <IndexRoute component={ChallengeScreenContainer} />
+        <Route path="challenges" component={ChallengeScreenContainer} />
```

Comment:
do we need both indexRoute and Route? so ChallengeScreenContainer is pointing to /main and /main/challenges